react/components/ExpandingTextArea/index.jsx

```diff
@@ -7,12 +7,12 @@ class ExpandingTextarea extends Component {
   }
 
   render () {
-    const { onChange, ...rest } = this.props;
+    const { ...rest } = this.props;
     return (
       <textarea
-        { ...rest }
+        {...rest}
         ref={x => this.el = x}
-        onChange={ this._handleChange.bind(this) }
+        onChange={this._handleChange.bind(this)}
```

Comment:
You should bind handlers in the component constructor instead of inside the render function

Reply:
The `xxx/index` was an accident; my IDE attempts to update paths when I move files between folders.  I fixed the handler binding as well.  Thanks!